src/mapping/parser/mod.rs

```diff
@@ -563,6 +563,13 @@ mod tests {
     #[test]
     fn check_parser() {
         let cases = vec![
+            (
+                ".v3ctor = \"bar\"",
+                Mapping::new(vec![Box::new(Assignment::new(
+                    "v3ctor".to_string(),
+                    Box::new(Literal::from(Value::from("bar"))),
+                ))]),
+            ),
```

Comment:
It would be worth adding a test to see what would happen if the path field name consisted soley of numbers to check if the path gets parsed as a path or a number first.

What would the following parse as:

```
.123 = 38
```

Reply:
It would parse as a path, since we don't support that float notation, you'd need to write `0.123` for it to be parsed as a float.

I'll add a test to validate that behaviour 👍